go-shfmt/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "3.7.0" %}
```

Comment:
can you add a comment to the conda-forge package for future reference?
https://anaconda.org/conda-forge/go-shfmt
https://github.com/conda-forge/go-shfmt-feedstock